packages/js/src/utils/npm-config.ts

```diff
@@ -108,13 +108,10 @@ async function getNpmConfigValue(key: string, cwd: string): Promise<string> {
 async function execAsync(command: string, cwd: string): Promise<string> {
   // Must be non-blocking async to allow spinner to render
   return new Promise<string>((resolve, reject) => {
-    exec(command, { cwd, windowsHide: false }, (error, stdout, stderr) => {
+    exec(command, { cwd, windowsHide: false }, (error, stdout) => {
```

Comment:
Is this change because sometimes warnings are printed to `stderr` and we wrongly reject the Promise? If that's the case, should we append/prepend the `stderr` to what we reject when `error` is defined and to what we resolve otherwise?

Reply:
there is an issue while i was debugging, the debugger output `'Debugger attached.Waiting for the debugger to disconnect...'` was passed in as stderr. i change the function to append to stderr and only to reject when exit code is 1.
`